tox.ini

```diff
@@ -37,15 +37,15 @@ deps =
     ruamel.yaml
     ansible
 commands =
-    bash -c 'if find tests/unit -name \*.py | grep -q . ; then pytest {posargs} tests/unit ; else echo no python files - skipping; fi'
+    bash -c 'if find tests/unit -name \*.py | grep -q . ; then mkdir -p /var/tmp/src/systemrole; mkdir -p /var/tmp/collections; COLLECTION_SRC_PATH=/var/tmp/src COLLECTION_DEST_PATH=/var/tmp/collections COLLECTION_ROLE=systemrole pytest {posargs} tests/unit; rm -rf /var/tmp/src /var/tmp/collections ; else echo no python files - skipping; fi'
```

Comment:
If a unit test fails, will this report a failure?

Reply:
Yes, I could see the failure in the travis-ci result page.
https://github.com/linux-system-roles/auto-maintenance/pull/13#issuecomment-694381047